components/ai_chat/resources/page/components/notices/notice_conversation_storage.tsx

```diff
@@ -51,7 +28,7 @@ export default function NoticeConversationStorage() {
           <a
             href='#'
             target='_blank'
-            onClick={() => aiChatContext.uiHandler?.openModelSupportUrl()}
+            onClick={() => aiChatContext.uiHandler?.openStorageSupportUrl()}
```

Comment:
++

Reply:
minor suggestion "Chat history is available in Brave version 1.75x and above" should add "on Desktop"

Reply:
...and android...

Reply:
so we should specify desktop and android unless we also think it's going to land in iOS 1.75x

Reply:
That edit will be made